### DISCUSSION TITLE
Sensor Watch Pro - Questions (and hopefully answers ;-))

### DISCUSSION THREAD

**Discussion by 814d3** [joeycastillo/Sensor-Watch#425] [General]
Hi all, I was very happy to read about Sensor Watch Pro Pre-Launch at [www.crowdsupply.com](https://www.crowdsupply.com/oddly-specific-objects/sensor-watch/updates/announcing-the-sensor-watch-pro-campaign-pre-launch)

Several questions came to my mind :-D

- Will the infrared light sensor be a add-on board to also use it with the original Sensor Watch?
- Is the amplified buzzer a software thing and could be easily adopted to the original Sensor Watch?
- does the accelerometer add-on board include the temperature components or are they placed at the mainboard like with Sensor Watch lite?

I like the LCD and will definitely order one :-)

**Comment by voloved** [comment]
Hey, I can answer these questions:

1. I haven't seen plans on an add-on board for the infrared sensor.
2. The amplified inductor requires a fairly-large inductor. It's not software-only and wouldn't be something the original will be able to have.
3. Accelerometer IC has an embedded temperature sensor, but the Pro also has a temperature sensor right on it.

**Comment by dvelazquez** [comment]
Has anyone tried powering the watch with a LIR2016 rechargeable battery instead of the CR2016?
The microcontroller voltage range is: 1.62V – 3.63V which is just in the edge of 3.6V-3.7V range of a LIR2016 battery.

**Reply by dvelazquez** [reply]
per SAML22 datasheet absolute maximum voltage rating is 3.8V
I don't see why it would not work with a LIR2016 battery...
